data/en/encodefor.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name":"encodeFor",
+	"type":"function",
+	"syntax":"encodeFor(type, value)",
+	"returns":"string",
+	"related":["encodeForHTML", "canonicalize"],
+	"description":"Encodes a given string for safe output in the specified context. The encoding in meant to mitigate Cross Site Scripting (XSS) attacks.",
```

Comment:
I think this is supposed to say "encoding _is_ meant to"

Reply:
PR #1077 issued.